content/asa/exercicios/0001-fichas-2021-2022.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Fichas (2021-2022)
+description: Resolução das fichas de ASA que estão disponíveis na página da UC (de 2021-2022).
+path: /asa/exercicios/fichas-2021-2022
+type: exercises
+---
+
+# Fichas (2021-2022)
+
+:::info[Resoluções Incorretas?]
+Caso encontres incorreções nas resoluções abaixo, por favor
+[reporta-as](https://github.com/diogotcorreia/resumos-leic/issues/new?assignees=&labels=incorrect+solution&template=incorrect_solution.yml&title=%5BIncorrect+Solution%5D+)
+para serem corrigidas.
+:::
+
+:::info[Mais Resoluções]
+Se quiseres adicionar as tuas resoluções, podes sempre fazer um [PR](https://github.com/diogotcorreia/resumos-leic)!
+:::
+
+- [Prática 1](https://drive.google.com/file/d/1o-mu3cpRvqNtj3tHnU8RifILAenDGWBR/view?usp=sharing)
+  :::::details[Implementação]
+  ::::tab-group
+  :::tab[Diogo Correia]
+  `embed:assets/main.cpp`
+  :::
+  :::tab[Diogo Gaspar]
+  `embed:assets/gaspa.cpp`
+  :::
+  :::tab[Luís Fonseca]
+  `embed:assets/luis.cpp`
+  :::
```

Comment:
The file path of these should be more specific, in order to allow to add more C++ solutions in the future.
I'll move them myself, don't worry.